contracts/ronin/slash-indicator/SlashRandomBeacon.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.9;
+
+import { HasContracts } from "../../extensions/collections/HasContracts.sol";
+import { IRoninValidatorSet } from "../../interfaces/validator/IRoninValidatorSet.sol";
+import { ISlashRandomBeacon } from "../../interfaces/slash-indicator/ISlashRandomBeacon.sol";
+import { ContractType } from "../../utils/ContractType.sol";
+
+abstract contract SlashRandomBeacon is ISlashRandomBeacon, HasContracts {
+  /// @dev value is equal to keccak256(abi.encode(uint256(keccak256("@ronin.SlashRandomBeacon.storage.SlashRandomBeaconConfig")) - 1)) & ~bytes32(uint256(0xff))
+  bytes32 private constant $$_SlashRandomBeaconConfigStorageLocation =
+    0x91c8f2da0132d5a54177c69679e2999120ba8f9ed42cabc35c99b69642ac8500;
+
+  /**
+   * @inheritdoc ISlashRandomBeacon
+   */
+  function slashRandomBeacon(address validatorId, uint256 period) external onlyContract(ContractType.RANDOM_BEACON) {
+    IRoninValidatorSet validatorContract = IRoninValidatorSet(getContract(ContractType.VALIDATOR));
+    SlashRandomBeaconConfig memory config = _getSlashRandomBeaconConfig();
+
+    emit Slashed(validatorId, SlashType.RANDOM_BEACON, period);
+
+    validatorContract.execSlash({
+      cid: validatorId,
+      newJailedUntil: 0,
+      slashAmount: config._slashAmount,
+      cannotBailout: true
```

Comment:
```suggestion
      cannotBailout: false
```

Reply:
check the cannot bailout in other slashes as well. Only unavailability tier-3 slashing is cannot bailout.